GiTinder/Services/GetProfilesForUser.cs

```diff
@@ -9,11 +9,11 @@
 
 namespace GiTinder.Services
 {
-    public class ProfileService
+    public class GetProfilesForUser
```

Comment:
@Riceqrisp the class can remain ProfileService

Reply:
Omg my bad. Corrected.